packages/webapp/src/components/Finances/PureFinanceTypeSelection/index.jsx

```diff
@@ -70,10 +70,12 @@ export default function PureFinanceTypeSelection({
           style={{ marginBottom: '24px' }}
         />
         <Main className={styles.leadText}>{leadText}</Main>
-        <Tiles
-          tileType={tileTypes.ICON_LABEL}
-          tileData={types}
-          formatTileData={getFormatTileDataFunc ? getFormatTileDataFunc(setValue) : formatTileData}
+        <List
+          listItemType={listItemType}
+          listItemData={types}
+          formatListItemData={
+            getFormatTileDataFunc ? getFormatTileDataFunc(setValue) : formatTileData
```

Comment:
Should we rename `getFormatTileDataFunc` and `formatTileData` since we no longer use tiles?

Reply:
This will always be a tile list in my heart, but since that name got veto-ed in the last PR, yes I guess they need to be renamed now!